src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The `All rights reserved.` is no longer part of the official header: https://docs.opensource.microsoft.com/content/releasing/copyright-headers.html

Reply:
fixed.